tools/llvm_toolchain.bzl

```diff
@@ -0,0 +1,185 @@
+"""
+Provides a wrapper around `llvm_toolchain` that uses a package provided sysroot
+instead of the system installed libraries.
+"""
+
+load(
+    "@bazel_toolchain//toolchain:deps.bzl",
+    _llvm_toolchain_dependencies = "bazel_toolchain_dependencies",
+)
+load(
+    "@bazel_toolchain//toolchain:rules.bzl",
+    _llvm_toolchain = "llvm_toolchain",
+    _llvm_toolchain_files = "llvm",
+)
+load("@local_config_info//:defs.bzl", "BAZEL_EXTERNAL_DIR")
+
+#  Rename to (hopefully) be less confusing
+llvm_toolchain_dependencies = _llvm_toolchain_dependencies
```

Comment:
why not just use the non-underscored name on line 8 and skip this?

Reply:
I believe it's not exposed otherwise.

Reply:
https://bazel.build/concepts/visibility#protecting-individual-symbols